include/common/core/freertos_task.hpp

```diff
@@ -73,4 +74,28 @@ class FreeRTOSTask {
     std::function<void()> starter{};
 };
 
+template <uint32_t StackDepth,
+          template <template <typename> typename> typename TaskObj>
```

Comment:
Is there a way to express this in a more easily understandable manner?

Reply:
Unfortunately I'm pretty sure there's not, but a comment should help a lot